### DISCUSSION TITLE
How to setup environmental variables with SvelteKit + Cloudflare Pages

### DISCUSSION THREAD

**Discussion by Pelps12** [prisma/prisma#15039] [Q&A]
I am trying to move my project to use prisma but I keep getting a "Datasource "db" references an environment variable "VITE_DATABASE_URL" that is not set" error even though I have it in my env file. I am using the edge client.  My build script is ``npm run build && prisma generate --data-proxy``. What could I be doing wrong?

**Comment by austincrim** [comment]
Hey there!

I believe SvelteKit changed how environment variables work in recent versions.

I would take a look at these docs specifically, where you import variables from `$env/static/private`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
